.env.local

```diff
@@ -4,3 +4,4 @@ WEBSITE_URL=http://localhost:3000
 API_URL=http://localhost:3060
 API_KEY=dvl-1510egmf4a23d80342403fb599qd
 LOG_LEVEL=debug
+OPENSSL_CONF=/dev/null
```

Comment:
Why?

Reply:
Sorry, this is a mistake which somehow I changed some months bank and forgot to revert. 🙉